conf/locale/uk/LC_MESSAGES/django.po

```diff
@@ -27478,14 +27478,12 @@ msgid "Certificate"
 msgstr "Сертифікат"
 
 #: lms/templates/certificates/_accomplishment-rendering.html
-msgid "This certificate is provided to"
```

Comment:
It is not a good idea to remove existing translations in case when we change translations just only for microsite.

Reply:
That translation I added a few time ago and now I just changed them